kolibri/core/assets/src/views/points-icon/index.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+
+  <div :style="dimensions">
+    <file-svg v-if="active" src="./icons/leaf.svg"/>
+    <file-svg v-else src="./icons/grey-leaf.svg"/>
+  </div>
+
+</template>
+
+
+<script>
+
+  function widthHeightValidate(units) {
+    return units.search(/[0-9]+(em|px|%)/) > -1;
+  }
+
+  module.exports = {
+    props: {
+      active: {
+        type: Boolean,
+        required: true,
+      },
+      height: {
+        type: String,
+        required: true,
+        validator: widthHeightValidate,
+      },
+      width: {
+        type: String,
+        required: true,
+        validator: widthHeightValidate,
+      },
```

Comment:
Yeah.. not actually sure why I did this either, because it works just fine doing it this way.